frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantsModal/VariantForm/VariantForm.tsx

```diff
@@ -88,6 +88,9 @@ const StyledFieldColumn = styled('div')(({ theme }) => ({
 
 const StyledInput = styled(Input)(() => ({
     width: '100%',
+    '& textarea': {
+        resize: 'vertical',
```

Comment:
I think setting a minimum height makes sense here, otherwise we can resize the input to be smaller than its original height.

Using `theme.spacing` in the `StyledInput`:

```TS
const StyledInput = styled(Input)(({ theme }) => ({
```

```suggestion
        minHeight: theme.spacing(3),
        resize: 'vertical',
```